ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -222,6 +222,10 @@ EDITOR_URL = 'http://localhost:8011/xep/initiate/'
 ICDS_SMS_INDICATOR_DOMAINS = ['{{ localsettings.ICDS_DOMAIN }}']
 {% endif %}
 
+{% if 'USER_REPORTING_METADATA_UPDATE_FREQUENCY' in localsettings %}
+USER_REPORTING_METADATA_UPDATE_FREQUENCY = ['{{ localsettings.USER_REPORTING_METADATA_UPDATE_FREQUENCY }}']
```

Comment:
This shouldn't be a list. And it might be easier to define it in hours in public.yml and just tack on ` * 60 * 60` here

Reply:
3ddd59b